python-client/trustedanalytics/core/files.py

```diff
@@ -638,3 +640,45 @@ def __init__(self, sql_query):
 
     def __repr__(self):
         return repr(self.file_name)
+
+
+
+class HBaseTable(object):
+    """
+    Define the query to retrieve the data from an hBase table.
+
+    Parameters
+    ----------
+    query : str
+        The sql query to retrieve the data
+
+    Returns
+    -------
+    class : HBaseTable object
+        An object which holds hBase query.
+
+    Examples
+    --------
+    .. code::
+
+        >>> import trustedanalytics as ta
+        >>> ta.connect()
+        >>> h = tp.HBaseTable ("my_table", [("pants", "aisle", unicode), ("pants", "row", int),( "shirts", "aisle", unicode),("shirts", "row", unicode)])
+        >>> f = tp.Frame(h)
+        >>> f.inspect()
+
+    """
+
+    def __init__(self, table_name, schema):
```

Comment:
Could we add start and end tag as optional params. This is small additional and almost a pass through.